macros/edr/data_monitoring/schema_changes/get_columns_snapshot_query.sql

```diff
@@ -0,0 +1,76 @@
+{% macro get_columns_snapshot_query(full_table_name) %}
+
+    {%- set known_columns_query %}
+        (
```

Comment:
These parentheses are redundant.

Reply:
Won't work without it, as I pass these queries as subqueries. 
But I agree it's ugly, I moved the parentheses to where these vars are called.